redis-dump.go

```diff
@@ -26,6 +26,24 @@ type stats struct {
 var client *redis.Client
 
 func init() {
+	// log level hierarchy anything that is info or above (debug, info, warn, error, fatal, panic). Default Info.
+	switch loglevel := os.Getenv("LOGLEVEL"); loglevel {
+		case "debug":
+			logrus.SetLevel(logrus.DebugLevel)
+		case "info":
+			logrus.SetLevel(logrus.InfoLevel)
+		case "warn":
+			logrus.SetLevel(logrus.WarnLevel)
+		case "error":
+			logrus.SetLevel(logrus.ErrorLevel)
+		case "fatal":
+			logrus.SetLevel(logrus.FatalLevel)
+		case "panic":
+			logrus.SetLevel(logrus.PanicLevel)
+		default:
+			logrus.SetLevel(logrus.InfoLevel)
+	}
+
```

Comment:
level, err := logrus.ParseLevel(os.Getenv("LOGLEVEL"))
logrus.SetLevel(level)

Use this instead of writing a switch case.